ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -48,6 +48,8 @@ objects:
     restartPolicy: Never
     activeDeadlineSeconds: 10800
     terminationGracePeriodSeconds: 900
+    securityContext:
+      privileged: true
```

Comment:
Do you remember why you needed this?

Reply:
I'm going to test without it, that was added when trying to get past the Permission Denied error for writing to .cache.. but I don't think it's required.. testing without it now..